i18n/fr/docusaurus-plugin-content-docs/version-25.10/graph-views/secure-your-map-platform.md

```diff
@@ -7,7 +7,7 @@ import TabItem from '@theme/TabItem';
 
 Ce chapitre décrit les procédures avancées permettant de sécuriser votre plateforme MAP.
 
-> Si vous souhaitez utiliser MAP en HTTPS, vous devez sécuriser à la fois votre plateforme Centreon et MAP. Suivez cette [procédure](../administration/secure-platform.md) pour sécuriser votre plateforme Centreon.
+> Si vous souhaitez utiliser MAP en HTTPS, vous devez sécuriser à la fois votre plateforme Centreon et MAP (ou MAP Legacy). Suivez cette [procédure](../administration/secure-platform.md) pour sécuriser votre plateforme Centreon.
```

Comment:
Since 24.10, Centreon MAP Legacy is not supported, so I think it's not necessary to quote here.